gensim/test/test_doc2vec.py

```diff
@@ -84,10 +84,11 @@ def test_int_doctags(self):
         """Test doc2vec doctag alternatives"""
         corpus = DocsLeeCorpus()
 
-        model = doc2vec.Doc2Vec(min_count=1)
+        size = 300
+        model = doc2vec.Doc2Vec(min_count=1, size=size)
```

Comment:
If these tests would succeed with the new (inherited) default size of 100, it'd be better to run them that way – and then test the internal array sizes for either matching '100' or (perhaps simply to verify internal self-consistency) 'model.vector_size'.